src/OpenSpaceNetArgs.cpp

```diff
@@ -439,183 +438,166 @@ void OpenSpaceNetArgs::printUsage(Action action) const
     cout << endl;
 }
 
+enum ArgUse {
```

Comment:
Interesting idea, it may be worth to make it more integral with `boost::program_options` in the future.